pkg/web3/types.go

```diff
@@ -22,6 +22,7 @@ type Web3Options struct {
 	MediationAddress  string `json:"mediation_address" toml:"mediation_address"`
 	JobCreatorAddress string `json:"jobcreator_address" toml:"jobcreator_address"`
 	PowAddress        string `json:"pow_address" toml:"pow_address"`
+	PowSubmitURL      string `json:"pow_submit_url" toml:"pow_submit_url"`
```

Comment:
let's rename to `PowBatchWsUrl`. Reason is, we want to make it generic if we end up batching other requests later on.